app/javascript/components/upload/UploadWizard.jsx

```diff
@@ -89,13 +89,10 @@ export function RawUploadWizard({ studyAccession, name }) {
 
 
   // used for toggling between the split view for the upload experiences
-  const [choiceMade, setChoiceMade] = useState(false)
+  const [overrideExperienceMode, setOverrideExperienceMode] = useState(false)
```

Comment:
For maintainability, let's either use:

```suggestion
  const [isOverrideExperienceMode, setIsOverrideExperienceMode] = useState(false)
```

like `const [isAnnDataExperience, setIsAnnDataExperience] = useState(false)`, or let's make the latter `const [annDataExperience, setAnnDataExperience] = useState(false)`.  

I personally think `isFoo` for ideal for booleans, but I don't feel strongly.  My only qualm here is with the locally inconsistent naming style.

Reply:
Oh in my mind I was reading 'override' as the action verb like override the experience mode? Yes or no. But I see how that could be confusing since 'override' could be read as a noun like is the override-experience-mode on or off? I can change it.